src/shared/seccomp-util.c

```diff
@@ -1619,7 +1620,6 @@ int seccomp_memory_deny_write_execute(void) {
                 case SCMP_ARCH_X86_64:
                 case SCMP_ARCH_X32:
                 case SCMP_ARCH_AARCH64:
-                case SCMP_ARCH_S390X:
                         filter_syscall = SCMP_SYS(mmap); /* amd64, x32, s390x, and arm64 have only mmap */
```

Comment:
comments needs minor updating

Reply:
Absolutely, good catch - dropping s390x from this.

I only wanted to wait with the push to see the result of the s390x CI run on Bionic to be able to rework it right away if there are still issues. But that Test now [completed](https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-bionic-upstream-systemd-ci-systemd-ci/bionic/s390x/s/systemd-upstream/20191204_124910_526f7@/log.gz) and worked.

Pushing with the comment cleanup now ...